apps/nextjs/components/SessionContext.tsx

```diff
@@ -69,7 +72,8 @@ export const SessionProvider: FC = (props) => {
   const value = {
     clearSession,
     email: _session?.email,
-    sessionId: _session?.sessionId,
+    isSessionIdValidated,
```

Comment:
My rationale was that this is descriptive in the places where the variable is going to be used. E.g. in `useFreshProps` something like `isReady` might cause some "what exactly is supposed to be ready?" kinda moments. Of course we can rename the variable in those places in case things get confusing.

Reply:
I'll leave the decision entirely up to you.

Basis for my suggestion is something I've seen a few projects, where the session keeps getting multiple similar `isSomething` flags which eventually end up being checked in downstream components as `isFoo && isBar` and those could've been optimized to a single `isReady` or such. A lot of devs follow existing conventions, and this naming kind of guides towards that direction IMO.